### PR TITLE
Readme paragraph about using Eclipse.

### PR DESCRIPTION
**Pull request by hovi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,4 +61,9 @@ Test scenarios also require Space Engineers running with the iv4XR plugin enable
...
 
 * Make sure you have installed [plugins](https://www.jetbrains.com/help/idea/enabling-cucumber-support-in-project.html#cucumber-plugin) `Gherkin` and `Cucumber for Java`
-* Right-click [.feature file](https://github.com/iv4xr-project/iv4xrDemo-space-engineers/tree/se-dev/src/jvmTest/resources/features) in IDEA and select "Run". 
\ No newline at end of file
+* Right-click [.feature file](https://github.com/iv4xr-project/iv4xrDemo-space-engineers/tree/se-dev/src/jvmTest/resources/features) in IDEA and select "Run".
+
+## Using Eclipse
+
+Eclipse does not have support for Kotlin multiplatform projects and so far we haven't been able to configure it to run with Kotlin JVM.
+We recommend using project with Jetbrains [IDEA](https://www.jetbrains.com/idea/download/).
```
Comment (PremekPaska) [inline README.md]: *the* project
JetBrains

File: README.md [hunk 2]
```diff
@@ -61,4 +61,9 @@ Test scenarios also require Space Engineers running with the iv4XR plugin enable
 For now, we run BDD tests from IDEA.
 
 * Make sure you have installed [plugins](https://www.jetbrains.com/help/idea/enabling-cucumber-support-in-project.html#cucumber-plugin) `Gherkin` and `Cucumber for Java`
-* Right-click [.feature file](https://github.com/iv4xr-project/iv4xrDemo-space-engineers/tree/se-dev/src/jvmTest/resources/features) in IDEA and select "Run". 
\ No newline at end of file
+* Right-click [.feature file](https://github.com/iv4xr-project/iv4xrDemo-space-engineers/tree/se-dev/src/jvmTest/resources/features) in IDEA and select "Run".
+
+## Using Eclipse
+
+Eclipse does not have support for Kotlin multiplatform projects and so far we haven't been able to configure it to run with Kotlin JVM.
```
Comment (PremekPaska) [inline README.md]: Could be simplified: 
Eclipse does not have support for -> Eclipse does not support

**Review by PremekPaska — APPROVED** [review]
Some minor fixes / improvements suggested.

File: README.md (hunk 1 above)
Comment (hovi) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (hovi) [reply README.md]: Done

**Pull request merged**